libs/platform/src/lib/components/value-help-dialog/components/define-tab/define-tab.component.html

```diff
@@ -0,0 +1,170 @@
+>
+    <div fd-panel-title>
+        <ng-container i18n="@@platformI18nValueHelpDialog.Define.ExcludeLabel">Exclude</ng-container> <ng-container *ngIf="_excluded.length">({{ _excluded.length }})</ng-container>
+    </div>
+    <div fd-panel-content>
+        <ng-container
+            [ngTemplateOutlet]="conditionContent"
+            [ngTemplateOutletContext]="{ items: _excluded, type: _defineTypes.exclude, filters: _excludeFilters, strategies: _excludeStrategy }"
+        ></ng-container>
+
+        <fd-layout-grid class="condition-row" *ngIf="_excluded.length === 0">
+            <div fdLayoutGridRow>
+                <div fdLayoutGridCol colGrow>&nbsp;</div>
+                <div fdLayoutGridCol class="action-col">
+                    <button fd-button i18n-aria-label="@@platformI18nValueHelpDialog.AddDefineCondition" aria-label="Add new define condition" glyph="add" (click)="addEmptyCondition(_defineTypes.exclude)"></button>
+                </div>
+            </div>
+        </fd-layout-grid>
+    </div>
+</fd-panel>
+
+
+<ng-template #conditionContent let-items="items" let-type="type" let-filters="filters" let-strategies="strategies">
+    <ng-container *ngFor="let item of items; trackBy: _trackByKeyAndType; index as i">
+        <fd-layout-grid class="condition-row">
+            <div fdLayoutGridCol="5">
+                <div fdLayoutGridRow>
+                    <fd-select
+                        class="vhd-condition-filters"
+                        fdLayoutGridCol
+                        colGrow
+                        i18n-placeholder="@@platformI18nValueHelpDialog.Define.Condition.FilterPlaceholder"
+                        placeholder="Select a filter from data source"
+                        [name]="uid + '-' + type + '-filter-' + item.id"
+                        [(ngModel)]="item.key"
+                        (ngModelChange)="_filterChanged()">
+                        <fd-option value="*">{{ fullBodyLabel }}</fd-option>
+                        <fd-option *ngFor="let filter of filters" [value]="filter.key">{{ filter.label }}</fd-option>
+                    </fd-select>
+                    <fd-select
+                        class="vhd-condition-filters"
+                        fdLayoutGridCol
+                        colGrow
+                        i18n-placeholder="@@platformI18nValueHelpDialog.Define.Condition.StrategyPlaceholder"
+                        placeholder="Select a strategy"
+                        [name]="uid + '-' + type + '-strategy-' + item.id"
+                        [(ngModel)]="item.strategy"
+                        (ngModelChange)="_filterChanged()">
+                        <fd-option *ngFor="let strategy of strategies" [value]="strategy.key">
+                            <ng-container *ngIf="type === 'exclude'" i18n="@@platformI18nValueHelpDialog.Define.Condition.StrategyPrefixExclude">not</ng-container>
+                            {{ strategy.label }}
+                        </fd-option>
```

Comment:
I'm afraid that's not going to work well with other languages.
`strategy.label` is not translatable and we can not translate "`not`" separately from other part of phrase, that should be one translation unit